movie-browser/src/index.css

```diff
@@ -1,10 +1,13 @@
-@tailwind base;
-
-@tailwind components;
-
-@tailwind utilities;
+.footer,
```

Comment:
are you still using `tailwind`? If not..remove it from the dependencies.

Reply:
Yes. Tailwind is the CSS framework in use alongside custom CSS. I've heard good things about it and figured this project was a good time to try it out.

Reply:
it's just that i saw u remove the imports. That's why i asked. But if you're still using it that's perfectly fine